scripts/data_collector/br_index/collector.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+import sys
+from pathlib import Path
+import importlib
+import datetime
+
+import fire
+import pandas as pd
+from tqdm import tqdm
+from loguru import logger
+
+CUR_DIR = Path(__file__).resolve().parent
+sys.path.append(str(CUR_DIR.parent.parent))
+
+from data_collector.index import IndexBase
+
+IBOV_INDEX_COMPOSITION = "https://raw.githubusercontent.com/igor17400/IBOV-HCI/main/historic_composition/{}.csv"
+
+YEAR_QUARTER = [
+    "2003_1Q",
+    "2003_2Q",
+    "2003_3Q",
+    "2004_1Q",
+    "2004_2Q",
+    "2004_3Q",
+    "2005_1Q",
+    "2005_2Q",
+    "2005_3Q",
+    "2006_1Q",
+    "2006_2Q",
+    "2006_3Q",
+    "2007_1Q",
+    "2007_2Q",
+    "2007_3Q",
+    "2008_1Q",
+    "2008_2Q",
+    "2008_3Q",
+    "2009_1Q",
+    "2009_2Q",
+    "2009_3Q",
+    "2010_1Q",
+    "2010_2Q",
+    "2010_3Q",
+    "2011_1Q",
+    "2011_2Q",
+    "2011_3Q",
+    "2012_1Q",
+    "2012_2Q",
+    "2012_3Q",
+    "2013_1Q",
+    "2013_2Q",
+    "2013_3Q",
+    "2014_1Q",
+    "2014_2Q",
+    "2014_3Q",
+    "2015_1Q",
+    "2015_2Q",
+    "2015_3Q",
+    "2016_1Q",
+    "2016_2Q",
+    "2016_3Q",
+    "2017_1Q",
+    "2017_2Q",
+    "2017_3Q",
+    "2018_1Q",
+    "2018_2Q",
+    "2018_3Q",
+    "2019_1Q",
+    "2019_2Q",
+    "2019_3Q",
+    "2020_1Q",
+    "2020_2Q",
+    "2020_3Q",
+    "2021_1Q",
+    "2021_2Q",
+    "2021_3Q",
+    "2022_1Q",
```

Comment:
Hi,
Why don't use a for loop to generate this list  until  `datetime.datetime.today()`

Reply:
Great!
Thanks for the suggestion
I'll work on this change